### PR TITLE
Feature/student homepage daniel

### PR DESCRIPTION
**Pull request by danieltantonio**
Just about done with Student Homepage feature. Ready for review.

### CODE REVIEW

File: src/components/pages/Headmaster/Mentees/MenteeDashDisplay.js [hunk 1]
```diff
@@ -8,7 +8,9 @@ import MenteeHome from './MenteeHome';
 const DashDisplayContainer = styled.div`
   z-index: 1;
   position: absolute;
-  left: 300px;
```
Comment (tompsherman) [inline src/components/pages/Headmaster/Mentees/MenteeDashDisplay.js]: good play making it dynamic

**Review by tompsherman — APPROVED** [review]

File: src/components/pages/Headmaster/Mentees/MenteeMenu.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    background-color: #4c4c49;
+    color: #b5b5b5;
+    transition: all 200ms linear;
+  }
+  width: 100%;
+`;
+
+export default function MenteeMenu(props) {
+  return (
+    <Drawer
```
Comment (stackpearson) [inline src/components/pages/Headmaster/Mentees/MenteeMenu.js:32]: love how configurable this is! Building that from scratch would be a chore.

File: src/components/pages/Headmaster/Mentees/MenteeHome.js [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+`;
+
+const CardContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 1%;
+  justify-content: space-evenly;
+`;
+
+export default function MenteeHome(props) {
```
Comment (stackpearson) [inline src/components/pages/Headmaster/Mentees/MenteeHome.js:21]: Are there any props actually being passed here or is that just a place holder? Based off the stakeholder meeting it didn't seem like this was a dynamic component, leaving that option open definetly doesn't hurt though!

**Review by stackpearson — DISMISSED** [review]

**Comment by abdimohamud** [comment]
Going to switch to this branch real quick and fix the potential merge conflict, has to do with App.js and how the student dashboard isn't pulled with PR.

File: src/index.js [hunk 4]
```diff
@@ -3,10 +3,11 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import { createStore, applyMiddleware } from 'redux';
+
 import thunk from 'redux-thunk';
 import App from './components/App';
 import 'antd/dist/antd.less';
-import reducer from './state/reducers/index';
```
Comment (abdimohamud) [inline src/index.js:9]: took out the index since theres an index.js file to grab from

**Review by abdimohamud — APPROVED** [review]

File: src/components/App.js [hunk 5]
```diff
@@ -46,7 +47,8 @@ const App = ({ role, checkToken }) => {
               {role === 'admin' && <AdminDashboard />}
 
               {/* checks token to see if role is mentee. credentials to check with -> mentees@mentees.com : password */}
-              {role === 'mentee' && <StudentDashboard />}
+              {role === 'student' && <StudentDashboard />}
+              {role === 'mentee' && <MenteeDashboard />}
```
Comment (abdimohamud) [inline src/components/App.js:51]: Here was the potential merge conflict, The Student Dashboard was using the mentee role (wasn't aware of it my bad), so I switched it to check for a student role. Will be working on the back-end to add student as a role and some seeds for it as well

**Comment by abdimohamud** [comment]
going to need another pull from main before merging to be up to date with recent pushes.

File: src/components/App.js (hunk 5 above)
Comment (tompsherman) [reply src/components/App.js:51]: seems like this is one of the miscommunication issues we've been having with stakeholder are similar words are being used interchangeably across technical conversations. I was under the impression initially mentees/students were the same, but after yesterday and the "program" role discussion it does seem like we  want a separate students dashboard for students to use personally and the mentee dashboard for the admin to view and edit as needed.

either way, good catch!

**Review by tompsherman — APPROVED** [review]

**Pull request merged**